index.d.ts

```diff
@@ -0,0 +1,15 @@
+import { GraphQLError } from "graphql";
+
+type JsonSchema = {
+  [key: string]: any;
```

Comment:
We are importing a plain `json` file I'm assuming this level of typing is enough. Am I right?

Reply:
Yeah, I'd think this is fine!